docset/winserver2022-ps/applocker/New-AppLockerPolicy.md

```diff
@@ -74,14 +87,41 @@ The existing AppLocker policy in the specified GPO will be overwritten.
 
 ## PARAMETERS
 
+### -AllowWindows
+Indicates that the AppLocker policy allows all local Windows components.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: FileInformation
```

Comment:
I am not sure if adding two yaml blocks under the parameter is the correct way or specifying

`Parameter Sets: (All)` 

like other cmdlet articles do, I do now that one specifies required true and the other false... @dariomws @tseward ?

Reply:
I agree. Maybe the best way is using `Parameter Sets: (All)`. And looking over the other example it seems that this parameter is not required.